### ISSUE TITLE
question about the illustration of features learned by SCL and BCL (figure 2. on supplementary material)

### ISSUE THREAD

**Issue by suminRoh** [FlamieZhu/BCL#12]
Thank you for your assistance! 
I have a question regarding Figure 2. 
In the figure, you illustrate loading features onto a 2D hypersphere, where the dimension of each feature is (1, feature dimension). Alternatively, if you are plotting logits on the 2D hypersphere, the dimensions of the logit is (1, 10) --> in cifar-10-lt.
 The description in Figure 2 mentions 'illustration of features,' but how are the features actually plotted onto the hypersphere?
